screens/TimeSelectorScreen.js

```diff
@@ -76,6 +98,24 @@ const styles = StyleSheet.create({
   container: {
     ...AppStyle.container
   },
+  picker: {
+    width: 300,
+    color: Colors.green500
+  },
+  pickerWrapper: {
+    ...AppStyle.outlinedButton,
+    padding: 0,
+    marginBottom: 20,
+    alignItems: 'stretch',
+    justifyContent: 'flex-start'
+
+  },
+  image: {
+    flex: 1,
+    width: undefined,
```

Comment:
is this necessary to sen 'undefined' to width and height

Reply:
yes, using 'flex: 1' and resizeMode: 'contain' sets image's size to fill remaining space but you have to unset image's original size to make it work